rust/stackable-cockpit/src/utils/k8s/client.rs

```diff
@@ -15,7 +15,7 @@ use snafu::{OptionExt, ResultExt, Snafu};
 use stackable_operator::{commons::listener::Listener, kvp::Labels};
 use tokio::sync::RwLock;
 use tracing::{Span, info, instrument};
-use tracing_indicatif::span_ext::IndicatifSpanExt as _;
+use tracing_indicatif::{indicatif_println, span_ext::IndicatifSpanExt as _};
```

Comment:
We should send errors to STDERR

```suggestion
use tracing_indicatif::{indicatif_eprintln, span_ext::IndicatifSpanExt as _};
```

Reply:
Was done as part of https://github.com/stackabletech/stackable-cockpit/pull/386/commits/d4691458099526018af8dfce8b3f2a1846f42dc7